mito-ai/src/Extensions/AiChat/ChatMessage/ScrollableSuggestions.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Saga Inc.
+ * Distributed under the terms of the GNU Affero General Public License v3.0 License.
+ */
+
+import React, { useEffect, useRef } from 'react';
+import { updateScrollMask, initScrollMask } from '../../../utils/scrollMask';
+
+interface SuggestionOption {
+    display: string;
+    prompt: string;
+}
+
+interface ScrollableSuggestionsProps {
+    options: SuggestionOption[];
+    onSelectSuggestion: (prompt: string) => void;
+}
+
+const ScrollableSuggestions: React.FC<ScrollableSuggestionsProps> = ({ 
+    options, 
+    onSelectSuggestion 
+}) => {
+    const suggestionsRef = useRef<HTMLDivElement>(null);
+
+    // Apply the scroll mask to the suggestions container when it mounts, updates, or is scrolled
+    useEffect(() => {
+        const suggestionsContainer = suggestionsRef.current;
+        if (!suggestionsContainer) return;
+        
+        // Initialize the mask with a delay to ensure content is rendered
+        initScrollMask(suggestionsContainer);
+        
+        // Update mask on scroll
+        const handleScroll = () => {
+            updateScrollMask(suggestionsContainer);
+        };
+        
+        // Update mask on window resize
+        const handleResize = () => {
+            updateScrollMask(suggestionsContainer);
+        };
+        
+        suggestionsContainer.addEventListener('scroll', handleScroll);
+        window.addEventListener('resize', handleResize);
+        
+        // Clean up the event listeners on unmount
+        return () => {
+            suggestionsContainer.removeEventListener('scroll', handleScroll);
+            window.removeEventListener('resize', handleResize);
+        };
+    }, [options.length]); // Re-apply when options change
+
+    return (
+        <div className="chat-suggestions" ref={suggestionsRef}>
```

Comment:
When my taskpane is wider, this looks a bit funky. Maybe we can center it? 

<img width="822" alt="Screenshot 2025-04-25 at 5 24 56 PM" src="https://github.com/user-attachments/assets/a382f1a3-ffac-42fd-a55b-d49121879978" />

Reply:
I created a new container with max width that is centered in the chat panel